spec/models/organization_spec.rb

```diff
@@ -17,6 +17,10 @@
     expect(Organization.new(donation_url: nil)).to have(1).error_on(:donation_url)
   end
 
+  it 'fails validation when no sort_order' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.